m4/check_pkg_nvtx.m4

```diff
@@ -9,15 +9,19 @@ AC_DEFUN([CHECK_PKG_NVTX], [
   check_pkg_found=yes
 
   check_pkg_CPPFLAGS_save="${CPPFLAGS}"
-  AC_ARG_WITH([nvtx],
-     [AS_HELP_STRING([--with-nvtx=DIR], [Enable tracing capability with NVTX @<:@default=no@:>@])])
+  AC_ARG_WITH([nvtx], [AS_HELP_STRING([--with-nvtx=DIR], [Path to NVTX headers.])])
```

Comment:
Bikeshedding, and not nearly as clear a help string.

Reply:
not sure what you mean by bikeshedding, my understanding of that term comes from [this freebsd mailing list lore](https://bikeshed.com/).

I can change this back. The point is that this change fixes the automatic enablement of nvtx, despite the help string previously saying it defaulted to "no".

Reply:
you made an unnecessary change in both formatting and the help string.  If you want to change the default, great.  But the rest is unnecessary noise.

Reply:
Please don't resolve issues until you've fixed the issue.